bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/search/VisitFormsSearchHandler.java

```diff
@@ -52,16 +54,22 @@ public PageableResult search(RequestContext context) throws ResponseException {
         String patientProgramUuid = context.getRequest().getParameter("patientProgramUuid");
         int numberOfVisits = Integer.parseInt(context.getRequest().getParameter("numberOfVisits"));
         String[] conceptNames = context.getRequest().getParameterValues("conceptNames");
+        Locale searchLocale = identifyLocale(context.getRequest().getSession().getAttribute("locale").toString());
 
         Patient patient = Context.getPatientService().getPatientByUuid(patientUuid);
         if (patient == null) {
             throw new InvalidSearchException("Patient does not exist.");
         }
         List<String> conceptNamesList = new ArrayList<>();
         if (conceptNames == null) {
-            conceptNamesList = getConcepts(Context.getConceptService().getConcept(ALL_OBSERVATION_TEMPLATES).getSetMembers());
+            List<Concept> concepts = Context.getConceptService().getConceptsByName(ALL_OBSERVATION_TEMPLATES, Locale.ENGLISH, null);
```

Comment:
the last param is whether to search for exact locale. In code, if this param is null, then its considered as false. So better to be explicit. Pass false, instead of null